trending_data/src/main/webapp/init_map.js

```diff
@@ -28,6 +28,11 @@ export async function initMap() {
   addAllMarkers(markerCluster);
   google.maps.event.addListener(map, 'click', function() {
     windowsHandler.currentWindow.close();
+    map.setOptions({styles: standard});
+    if (map.freeze_when_popup_is == true) {
```

Comment:
A few things - you should declare this variable somewhere and comment what it does. Then I would give a more descriptive name (this name is not correct English - maybe `freeze_when_popup_is_open`?). And finally, as a nit - you can drop the `== true` and just test `if (map.freeze_when_popup_is_open) {`.